docs/readthedocs/source/doc/PPML/Overview/ppml.md

```diff
@@ -222,39 +222,177 @@ The result should look like this:
+    -Xmx8g \
+    org.apache.spark.deploy.SparkSubmit \
+    --master $RUNTIME_SPARK_MASTER \
+    --deploy-mode $SPARK_MODE \
+    --name spark-pi-sgx \
+    --conf spark.driver.host=$SPARK_LOCAL_IP \
+    --conf spark.driver.port=$RUNTIME_DRIVER_PORT \
+    --conf spark.driver.memory=$RUNTIME_DRIVER_MEMORY \
+    --conf spark.driver.cores=$RUNTIME_DRIVER_CORES \
+    --conf spark.executor.cores=$RUNTIME_EXECUTOR_CORES \
+    --conf spark.executor.memory=$RUNTIME_EXECUTOR_MEMORY \
+    --conf spark.executor.instances=$RUNTIME_EXECUTOR_INSTANCES \
+    --conf spark.kubernetes.authenticate.driver.serviceAccountName=spark \
+    --conf spark.kubernetes.container.image=$RUNTIME_K8S_SPARK_IMAGE \
+    --conf spark.kubernetes.driver.podTemplateFile=/ppml/trusted-big-data-ml/spark-driver-template.yaml \
+    --conf spark.kubernetes.executor.podTemplateFile=/ppml/trusted-big-data-ml/spark-executor-template.yaml \
+    --conf spark.kubernetes.executor.deleteOnTermination=false \
+    --conf spark.network.timeout=10000000 \
+    --conf spark.executor.heartbeatInterval=10000000 \
+    --conf spark.python.use.daemon=false \
+    --conf spark.python.worker.reuse=false \
+    --conf spark.kubernetes.sgx.enabled=$SGX_ENABLED \
+    --conf spark.kubernetes.sgx.driver.mem=$SGX_DRIVER_MEM \
+    --conf spark.kubernetes.sgx.driver.jvm.mem=$SGX_DRIVER_JVM_MEM \
+    --conf spark.kubernetes.sgx.executor.mem=$SGX_EXECUTOR_MEM \
+    --conf spark.kubernetes.sgx.executor.jvm.mem=$SGX_EXECUTOR_JVM_MEM \
+    --conf spark.kubernetes.sgx.log.level=$SGX_LOG_LEVEL \
+    --conf spark.authenticate=true \
+    --conf spark.authenticate.secret=$secure_password \
+    --conf spark.kubernetes.executor.secretKeyRef.SPARK_AUTHENTICATE_SECRET="spark-secret:secret" \
+    --conf spark.kubernetes.driver.secretKeyRef.SPARK_AUTHENTICATE_SECRET="spark-secret:secret" \
+    --conf spark.authenticate.enableSaslEncryption=true \
+    --conf spark.network.crypto.enabled=true \
+    --conf spark.network.crypto.keyLength=128 \
+    --conf spark.network.crypto.keyFactoryAlgorithm=PBKDF2WithHmacSHA1 \
+    --conf spark.io.encryption.enabled=true \
+    --conf spark.io.encryption.keySizeBits=128 \
+    --conf spark.io.encryption.keygen.algorithm=HmacSHA1 \
+    --conf spark.ssl.enabled=true \
+    --conf spark.ssl.port=8043 \
+    --conf spark.ssl.keyPassword=$secure_password \
+    --conf spark.ssl.keyStore=/ppml/trusted-big-data-ml/work/keys/keystore.jks \
+    --conf spark.ssl.keyStorePassword=$secure_password \
+    --conf spark.ssl.keyStoreType=JKS \
+    --conf spark.ssl.trustStore=/ppml/trusted-big-data-ml/work/keys/keystore.jks \
+    --conf spark.ssl.trustStorePassword=$secure_password \
+    --conf spark.ssl.trustStoreType=JKS \
+    --class org.apache.spark.examples.SparkPi \
+    --verbose \
+    local:///ppml/trusted-big-data-ml/work/spark-3.1.2/examples/jars/spark-examples_2.12-3.1.2.jar 100 2>&1 | tee spark-pi-sgx-$SPARK_MODE.log
+```
+

```

Comment:
We can add a string output (`kubectl get po`). to Tell user, what they can get.
Tell them, they can get output with `kubectl logs -f driver-xxxx`

Reply:
update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R321